src/shared/components/button/button.tsx

```diff
@@ -0,0 +1,28 @@
+import type { ButtonHTMLAttributes, ReactNode } from "react";
+
+import { root, sizeStyles, variantStyles } from "./button.css";
```

Comment:
css 스타일은
```suggestion
import * as styles from "./button.css"
```
이렇게 한번에 불러와서 사용해주세요!! `컨벤션!!`